apps/pie-docs/src/_11ty/shortcodes/index.js

```diff
@@ -10,6 +10,7 @@ const link = require('./link');
 const list = require('./list');
 const mediaElement = require('./mediaElementList');
 const notification = require('./notification');
+const componentCodeMoreInformationNotification = require('./componentCodeMoreInformationNotification');
```

Comment:
I won't make you do this too but if you need to make any other changes then this can be alphabetised while you're at it